core/src/main/java/org/kiwix/kiwixmobile/core/dao/NewBookmarksDao.kt

```diff
@@ -21,24 +21,15 @@ import io.objectbox.Box
 import io.objectbox.kotlin.query
 import io.reactivex.Flowable
 import io.reactivex.schedulers.Schedulers
-import org.kiwix.kiwixmobile.core.bookmark.BookmarkItem
+import org.kiwix.kiwixmobile.core.page.bookmark.adapter.BookmarkItem
 import org.kiwix.kiwixmobile.core.dao.entities.BookmarkEntity
 import org.kiwix.kiwixmobile.core.dao.entities.BookmarkEntity_
 import org.kiwix.kiwixmobile.core.data.local.entity.Bookmark
 import org.kiwix.kiwixmobile.core.reader.ZimFileReader
 import javax.inject.Inject
 
 class NewBookmarksDao @Inject constructor(val box: Box<BookmarkEntity>) {
-  fun getBookmarks(
-    fromCurrentBook: Boolean,
-    zimFileReader: ZimFileReader?
-  ) = box.query {
-    if (fromCurrentBook) {
-      equal(BookmarkEntity_.zimName, zimFileReader?.name ?: "")
-    }
-    order(BookmarkEntity_.bookmarkTitle)
```

Comment:
they used to be ordered by title with this query, we may wish to restore that functionality